README.md

```diff
@@ -11,6 +11,51 @@ The backend for a generic mobile-friendly quiz engine created using FastAPI and
 
 ### Local DB Setup
 
+
+- The following steps are for a Linux.
+- To run the backend locally, you would need to setup a local instance of mongodb. The offical instructions [here](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-ubuntu/) are good and simple enough to follow. Those steps are also listed down below.
+
+  - Import the public key used by the package management system. From a terminal, install gnupg if it is not already available:
+    ```
+    sudo apt-get install gnupg
+    ```
```

Comment:
here you have written two things. one is importing the public key and then you've written to install GnuPG.
But you've only added the command for GnuPG installation. What about the public key?